libraries/chain/steem_evaluator.cpp

```diff
@@ -778,11 +784,14 @@ namespace golos { namespace chain {
                     uint16_t reward_weight = comment_bandwidth{_db, now, o, mprops, auth}.calc_reward_weight();
 
                     db().modify(auth, [&](account_object &a) {
-                        a.last_post = now;
                         if (o.parent_author != STEEMIT_ROOT_POST_PARENT) {
                             a.comment_count++;
+                            a.last_post = now;
```

Comment:
Looks strange. 
In case of comments: we are set last_post in any case
In case of posts: we are don't change any statistic field.

Reply:
@afalaleev For posts we do not need to change any field in account because we have it in bandwidth. `last_post` is need only as last comment time, not post.

Reply:
Add field `last_comment`, it should contain time of last comment.
Save in `last_post` the time of last post.

We already make such modifications: as an example, check changes arround `comment_count` and `post_count`

Reply:
@afalaleev But we already have account_bandwidth_object, why waste memory - think that I didn't understood and made this pull-request.